src/ExpenseTracker.Common/Model/BaseModel.cs

```diff
@@ -1,8 +1,32 @@
+using System;
+using ExpenseTracker.Common.Constants;
+
 namespace ExpenseTracker.Common.Model
 {
-    public class BaseModel
+    public abstract class BaseModel : IBaseModel
     {
         public long Id { get; set; }
-        public string Status { get; set; } = "Active";
+        public DateTime CreatedDate { get; set; } = DateTime.Now;
+        public string Status { get; set; } = StatusConstants.StatusActive;
```

Comment:
@BirajMainali Letting users to set Status arbitrarily might not be a good idea. We should make these properties `protected` in the base model.